bridges/UnogsBridge.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+class UnogsBridge extends BridgeAbstract {
+
+	const MAINTAINER = 'csisoap';
+	const NAME = 'uNoGS Bridge';
+	const URI = 'https://unogs.com';
+	const DESCRIPTION = 'Return what\'s new or removal on Netflix';
+
+	const PARAMETERS = array(
+		'global' => array(
+			'feed' => array(
+				'name' => 'feed',
+				'type' => 'list',
+				'title' => 'Choose whether you want latest movies or removal on Netflix',
+				'values' => array(
+					'What\'s New' => 'new last 7 days',
+					'Expiring' => 'expiring'
+				)
+			)
+		),
+		'Global' => array(),
+		'Country' => array(
+			'country_code' => array(
+				'name' => 'Country',
+				'type' => 'list',
+				'title' => 'Choose your preferred country',
+				'values' => array(
+					'Argentina' => 21,
+					'Australia' => 23,
+					'Belgium' => 26,
+					'Brazil' => 29,
+					'Canada' => 33,
+					'Colombia' => 36,
+					'Czech Republic' => 307,
+					'France' => 45,
+					'Germany' => 39,
+					'Greece' => 327,
+					'Hong Kong' => 331,
+					'Hungary' => 334,
+					'Iceland' => 265,
+					'India' => 337,
+					'Israel' => 336,
+					'Italy' => 269,
+					'Japan' => 267,
+					'Lithuania' => 357,
+					'Malaysia' => 378,
+					'Mexico' => 65,
+					'Netherlands' => 67,
+					'Philippines' => 390,
+					'Poland' => 392,
+					'Portugal' => 268,
+					'Romania' => 400,
+					'Russia' => 402,
+					'Singapore' => 408,
+					'Slovakia' => 412,
+					'South Africa' => 447,
+					'South Korea' => 348,
+					'Spain' => 270,
+					'Sweden' => 73,
+					'Switzerland' => 34,
+					'Thailand' => 425,
+					'Turkey' => 432,
+					'Ukraine' => 436,
+					'United Kingdom' => 46,
+					'United States' => 78
+				)
+			)
+		)
+	);
+
+	public function getName() {
+		$feedName = '';
+		if($this->queriedContext == 'Global') {
+			$feedName .= 'Netflix Global - ';
+		} elseif($this->queriedContext == 'Country') {
+			$feedName .= 'Netflix Country Code: ' . $this->getInput('country_code') . ' - ';
+		}
+		if($this->getInput('feed') == 'expiring') {
+			$feedName .= 'Expiring title';
+		} elseif($this->getInput('feed') == 'new last 7 days') {
+			$feedName .= 'What\'s New';
+		} else {
+			$feedName = self::NAME;
+		}
+		return $feedName;
+	}
+
+	private function getJSON($url) {
+		$header = array(
+			'Referer: https://unogs.com/',
+			'referrer: http://unogs.com'
+		);
+
+		$raw = getContents($url, $header)
+				or returnServerError('Unable to get data from uNoGS');
```

Comment:
or returnServerError is useless. getContents will raise the error.

Reply:
@em92 it's done